parser_library/src/workspaces/parse_lib_provider.h

```diff
@@ -41,7 +41,7 @@ class parse_lib_provider
 
     virtual std::optional<std::string> get_library(const std::string& library,
         const utils::resource::resource_location& program,
-        std::optional<utils::resource::resource_location>* file_loc) const = 0;
+        std::optional<utils::resource::resource_location>& lib_location) const = 0;
```

Comment:
ok, slight misunderstanding here - I meant that the function should not use empty string as an indicator that the value is not available... not that the whole parameter is mandatory now.

Reply:
I guess there was no misunderstanding as I've removed it deliberately because no one was using `lib_location` as optional. I can revert it, of course

Reply:
ok, keep it then.